metabrainz/templates/index/code-of-conduct.html

```diff
@@ -42,6 +42,10 @@ <h1 class="page-title">{{ _('Code of Conduct') }}</h1>
       Use gender neutral language whenever possible. Unless actually talking about genders or specific people, there is no reason 
       to use gendered language. As we touch older work that is not gender neutral, we should make an effort to make it gender neutral.
     </li>
+    <li>
+      Don’t abuse or game the systems. Use of <a href="https://en.wikipedia.org/wiki/Sockpuppet_(Internet)">sockpuppets</a>, unauthorised bots,
```

Comment:
The wording of this doesn't fit with the usual tone of voice we use in our projects. Let's try something softer that fits with our usual messaging:

"Do not abuse or game any of our projects. All of our projects are community maintained and abusive actions negatively affect other users. Do not use or create <a href="https://en.wikipedia.org/wiki/Sockpuppet_(Internet)">sockpuppets</a>, unauthorised bots, or any other means of circumventing the intent or integrity of our projects.

Reply:
I'll fix this one thing after merging, is easier.